src/client/lazy-app/Compress/index.tsx

```diff
@@ -377,6 +377,7 @@ export default class Compress extends Component<Props, State> {
 
   componentWillUnmount(): void {
     updateDocumentTitle({ loading: false });
+    this.widthQuery.removeListener(this.onMobileWidthChange);
```

Comment:
This doesn't seem to be causing a leak. Or, if it is, it's small. But, good to add it for safety.

Reply:
(I was wrong here, it was causing a leak, it was just hard to see due to the resizeObserver issue)